### PR TITLE
FLUT-862867-[feature][flutter]: Add content for 25.1 new features

### PR DESCRIPTION
**Pull request by immankumarsync**
Added content for 25.1 new features

### CODE REVIEW

File: Flutter/pdf-viewer/form-filling.md [hunk 1]
```diff
@@ -668,6 +668,72 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
+## Undo, redo actions on Form fields
```
Comment (ramkumarsync) [inline Flutter/pdf-viewer/form-filling.md:671]: Undo and redo actions on form fields

File: Flutter/pdf-viewer/form-filling.md [hunk 2]
```diff
@@ -668,6 +668,72 @@ Widget build(BuildContext context) {
...
+
+If you performed any undesired actions when editing the form fields, you can undo and redo the action to restore the previous state. This section will go through how to undo and redo the changes made to the form fields.
```
Comment (ramkumarsync) [inline Flutter/pdf-viewer/form-filling.md:673]: If you performed any undesired actions when editing the form fields, you can undo and redo the action to restore the previous state.

The undo and redo operations are performed by assigning the `UndoHistoryController` instance to the `undoController` property of the `SfPdfViewer`. The `UndoHistoryController` class contains the `undo` and 'redo` methods to perform the same, respectively. The `canUndo` and `canRedo` properties are used to check whether the undo and redo operations can be performed or not, respectively. The following code example illustrates how to perform the form filling undo and redo operations programmatically with the `SfPdfViewer`.

// code

For desktop platforms such as Windows, macOS, and desktop web, you can use the following shortcut keys to perform the actions:
 
Note: This `undoController` is common for both annotations and form fields. You do not need to create separate instances for both.

File: Flutter/pdf-viewer/getting-started.md [hunk 3]
```diff
@@ -160,6 +160,26 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
+## Load document with the specified page
```
Comment (ramkumarsync) [inline Flutter/pdf-viewer/getting-started.md:163]: Load the document with the specified page

File: Flutter/pdf-viewer/magnification.md [hunk 4]
```diff
@@ -71,6 +71,10 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
+### Active view port rendering at higher zoom percentages
+
+From the 25.1 version, the [maxZoomLevel](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/SfPdfViewer/maxZoomLevel.html) of the PDF Viewer can be extended with the support of active view port rendering at higher zoom percentages. It renders only the part of the PDF file that is visible on-screen and ignoring the parts that are outside the viewport. The mode is automatically enabled when the page size or zoom increased beyond a specified limit on the zooming. This approach will be helpful to open the large-size pages containing PDF documents at higher zoom levels.
```
Comment (ramkumarsync) [inline Flutter/pdf-viewer/magnification.md:76]: With the support of active viewport rendering, at higher zoom levels, only the parts of the PDF file that are visible on-screen are rendered, ignoring the parts that are outside the viewport. The mode is automatically enabled when the page size or zoom increases beyond the zoom level of 2. This approach will be helpful to open large-size pages containing PDF documents at higher zoom levels.

File: Flutter/pdf-viewer/magnification.md [hunk 5]
```diff
@@ -71,6 +71,10 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
+### Active view port rendering at higher zoom percentages
```
Comment (ramkumarsync) [inline Flutter/pdf-viewer/magnification.md:74]: Active viewport rendering at higher zoom levels

**Review by ramkumarsync — APPROVED** [review]

File: Flutter/pdf-viewer/form-filling.md [hunk 6]
```diff
@@ -668,6 +668,74 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
+## Undo and redo actions on form fields
+
+If you performed any undesired actions when editing the form fields, you can undo and redo the action to restore the previous state.
```
Comment (Chrispineagunja) [inline Flutter/pdf-viewer/form-filling.md:673]: If you performed undesired actions when editing the form fields, you can undo and redo the action to restore the previous state.

File: Flutter/pdf-viewer/form-filling.md [hunk 7]
```diff
@@ -668,6 +668,74 @@ Widget build(BuildContext context) {
...
+        undoController: _undoController,
+      ),
+    ),
+  );
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+For desktop platforms such as Windows, macOS, and desktop web, you can use the following shortcut keys to perform the actions.
```
Comment (Chrispineagunja) [inline Flutter/pdf-viewer/form-filling.md:717]: For desktop platforms such as Windows, macOS, and desktop web, use the following shortcut keys to perform the actions.

File: Flutter/pdf-viewer/form-filling.md [hunk 8]
```diff
@@ -668,6 +668,74 @@ Widget build(BuildContext context) {
...
+<td><code>Command</code> + <code>z</code></td>
+</tr>
+<tr>
+<th>Redo</th>
+<td><code>Ctrl</code> + <code>y</code></td>
+<td><code>Command</code> + <code>y</code></td>
+</tr>
+</table>
+
+N> This `undoController` is common for both annotations and form fields. You do not need to create separate instances for both.
```
Comment (Chrispineagunja) [inline Flutter/pdf-viewer/form-filling.md:737]: N> This `undoController` is common for annotations and form fields. You do not need to create separate instances for both.

File: Flutter/pdf-viewer/getting-started.md [hunk 9]
```diff
@@ -160,6 +160,26 @@ Widget build(BuildContext context) {
...
+  return Scaffold(
+      body: SfPdfViewer.network(
+              'https://cdn.syncfusion.com/content/PDFViewer/flutter-succinctly.pdf',
+              initialPageNumber: 5));
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+N> It is recommended not to use both the [initialScrollOffset](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/SfPdfViewer/initialScrollOffset.html) and `initialPageNumber` properties at the same time. If both properties are defined, then the `initialPageNumber` will be prioritized over the [initialScrollOffset](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/SfPdfViewer/initialScrollOffset.html)
```
Comment (Chrispineagunja) [inline Flutter/pdf-viewer/getting-started.md:181]: N> It is recommended not to use both the [initialScrollOffset](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/SfPdfViewer/initialScrollOffset.html) and `initialPageNumber` properties at the same time. If both properties are defined, the `initialPageNumber` will be prioritized over the [initialScrollOffset](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/SfPdfViewer/initialScrollOffset.html)

File: Flutter/pdf-viewer/magnification.md [hunk 10]
```diff
@@ -71,6 +71,10 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
+### Active viewport rendering at higher zoom levels
+
+With the support of active viewport rendering, at higher zoom levels, only the parts of the PDF file that are visible on-screen are rendered, ignoring the parts that are outside the viewport. The mode is automatically enabled when the page size or zoom increases beyond the zoom level of 2. This approach will be helpful to open large-size pages containing PDF documents at higher zoom levels.
```
Comment (Chrispineagunja) [inline Flutter/pdf-viewer/magnification.md:76]: With the support of active viewport rendering, at higher zoom levels, only the parts of the PDF file that are visible on-screen are rendered, ignoring the parts outside the viewport. The mode is automatically enabled when the page size or zoom increases beyond the zoom level of 2. This approach will be helpful to open large-size pages containing PDF documents at higher zoom levels.

File: Flutter/pdf-viewer/undo-redo.md [hunk 11]
```diff
@@ -31,7 +31,7 @@ For desktop platforms such as Windows, macOS, and desktop web, you can use the f
...
 
-You can perform the undo and redo operations in the `SfPdfViewer` by assigning the `UndoHisotyController` instance to the `undoController` property of the `SfPdfViewer`. The UndoHistoryController class contains the `undo` and `redo` methods to perform the undo and redo operations, respectively. The `canUndo` and `canRedo` properties are used to check whether the undo and redo operations can be performed, respectively. The following code example illustrates how to perform the undo and redo operations programmatically in the `SfPdfViewer` with the help of the UndoHistoryController class.
+You can perform the undo and redo operations in the `SfPdfViewer` by assigning the `UndoHistoryController` instance to the `undoController` property of the `SfPdfViewer`. The UndoHistoryController class contains the `undo` and `redo` methods to perform the undo and redo operations, respectively. The `canUndo` and `canRedo` properties are used to check whether the undo and redo operations can be performed, respectively. The following code example illustrates how to perform the undo and redo operations programmatically in the `SfPdfViewer` with the help of the UndoHistoryController class.
```
Comment (Chrispineagunja) [inline Flutter/pdf-viewer/undo-redo.md:34]: You can perform the undo and redo operations in the `SfPdfViewer` by assigning the `UndoHistoryController` instance to the `undoController` property of the `SfPdfViewer`. The UndoHistoryController class contains the `undo` and `redo` methods to perform the undo and redo operations, respectively. The `canUndo` and `canRedo` properties check whether the undo and redo operations can be performed, respectively. The following code example illustrates how to perform the undo and redo operations programmatically in the `SfPdfViewer` with the help of the UndoHistoryController class.

File: Flutter/pdf-viewer/undo-redo.md [hunk 12]
```diff
@@ -71,4 +71,6 @@ Widget build(BuildContext context) {
 }
 
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+N> This `undoController` is common for both annotations and form fields. You do not need to create separate instances for both.
```
Comment (Chrispineagunja) [inline Flutter/pdf-viewer/undo-redo.md:76]: N> This `undoController` is common for annotations and form fields. You do not need to create separate instances for both.

**Review by Chrispineagunja — COMMENTED** [review]
Content review completed

**Pull request merged**